tests/shell/check_error.sh

```diff
@@ -91,7 +91,7 @@ if is_plugin_available error; then
 	#
 	#CONTENT=`cat $TMPFILE`
```

Comment:
This is still open (can be done in separate PR)

Reply:
I remember that you wanted me to 'cleanup' the shell script but how exactly do you want me to do that? Should I remove the commented code?

Reply:
The idea is of course to make the commented-out code work again.